lib/miq_expression.rb

```diff
@@ -3,15 +3,17 @@ class MiqExpression
   include Vmdb::Logging
   attr_accessor :exp, :context_type, :preprocess_options
 
+  # rubocop:disable Security/YAMLLoad
   config = YAML.load(ERB.new(File.read(Rails.root.join("config", "miq_expression.yml"))).result)
+  # rubocop:enable Security/YAMLLoad
   BASE_TABLES = config[:base_tables]
   INCLUDE_TABLES = config[:include_tables]
   EXCLUDE_COLUMNS = config[:exclude_columns]
   EXCLUDE_EXCEPTIONS = config[:exclude_exceptions]
   TAG_CLASSES = config[:tag_classes]
   EXCLUDE_FROM_RELATS = config[:exclude_from_relats]
   FORMAT_SUB_TYPES = config[:format_sub_types]
-  FORMAT_BYTE_SUFFIXES = FORMAT_SUB_TYPES[:bytes][:units].inject({}) { |h, (v, k)| h[k] = v; h }
+  FORMAT_BYTE_SUFFIXES = FORMAT_SUB_TYPES[:bytes][:units].each_with_object({}) { |(v, k), h| h[k] = v }
```

Comment:
`invert` is nice if you are sure that your values are also unique (which in this case, we know they are because the original code was essentially doing the same thing)